docker/build_test.sh

```diff
@@ -547,6 +547,13 @@ function test_build_with_tag() {
     "[\"gcr.io/build/with:tag\"]"
 }
 
+function test_build_with_passwd() {
+  # We should have a tag in our manifest containing the name
+  # specified via the tag kwarg.
+  check_layers "with_passwd" \
+    "621fe329a78d65d90d34d6dc277ccac2249bba4c8228222271418bf6b07c4dec"
+}
```

Comment:
You should add an additional check that uses `get_layer_listing` to check that the `passwd` file is in the new layer.